app/Models/User.php

```diff
@@ -1814,6 +1814,28 @@ public function titleUrl(): ?string
         return $this->rank?->url;
     }
 
+    public function toMetaDescription(array $options = []): ?string
+    {
+        $ruleset = $options['ruleset'] ?? $this->playmode;
+        $stats = $this->statistics($ruleset);
+        if ($stats === null) {
+            return null;
+        }
+
+        $countryRank = $stats->countryRank();
+        $globalRank = $stats->globalRank();
```

Comment:
I wonder if there should be something about variants

Reply:
I omitted the variants to keep the description simple.

Also, figuring out how to remove the `where` query from the transformer so it doesn't always do a separate query 🤔

Reply:
can probably just be updated to use the now-updated `statistics(...)` and preload as usual

Reply:
`UserStatisticsTransformer` doesn't use `User`, though and preloading a relation on `User` for the transformer to go through seems kind of obtuse? 🤔

Reply:
I mean preloaded through wherever it's being used. or in case of single user use the non query version of it so it'll be cached on next usage `$user->statistics('mania', variant: $variant)`